pages/pipelines/block_step.md

```diff
@@ -102,8 +102,9 @@ Optional attributes:
   <tr>
     <td><code>key</code></td>
     <td>
-	    <p>A unique string to identify the block step.</p>
+      <p>A unique string to identify the block step.<br>Keys can not have the same pattern as a UUID (<code>xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</code>).</p>
```

Comment:
Not sure why `bloack_step.md` uses `<p>` for all the table cells, but none of the others do 🤔

Reply:
Good question @thejcannon , and thanks for pointing this out. I'm not sure why this is the case either.

I'll remove these `<p></p>` elements from this table then.